sqlite/src/random.c

```diff
@@ -21,7 +21,7 @@
 /* All threads share a single random number generator.
 ** This structure is the current state of the generator.
 */
-static SQLITE_WSD struct sqlite3PrngType {
+static __thread SQLITE_WSD struct sqlite3PrngType {
   unsigned char isInit;          /* True if initialized */
```

Comment:
To avoid future merge conflicts, can we wrap this in the normal #if blocks for SQLite customizations?

Reply:
I'm assuming upstream SQLite doesn't have this bug (due to its locking vs ours?)

Reply:
Right, canonical SQLite is immune to this issue.